packages/apps/workshop/stories/components/box.stories.js

```diff
@@ -16,16 +16,16 @@ storiesOf('Old|Styles/Box', module)
         ultricies lacus in, fermentum nisi. Cras sed neque id dui
         fermentum faucibus. Mauris mollis arcu justo, eget convallis
         massa volutpat ac.
-      </p>
+      </div>
     </div>
     `,
   )
   .add(
     'Light',
     () => `
     <div class="oui-box oui-box_light">
-      <h1>An awesome title for an awesome box</h1>
-      <p>
+      <div class="oui-box__heading">An awesome title for an awesome box</div>
+      <div class="oui-box__body">
```

Comment:
What if I put a paragraph or a list in the box ?
I think it's a bad idea to have specific text style this component. The style provided by the base tags should be sufficient.

Reply:
ODS' styling shows different font colors between heading and content. I'm completely aligned with the fact that the box itself should have a 'default' text styling, but heading has to be treated differently.

Reply:
I don't have any problem with `oui-box__heading`. We've already talked about that :)
By talking a bit with CX, the heading should always have this size. So it's the right thing to do (but need to be a `<h*>` tag).

But my comment was more focused on `oui-box__body` that is useless here. We need to change our base text and heading colors first (https://zeroheight.com/3a97bbddb/p/3233cd/t/40105e) ;)